plotting/comparison.py

```diff
@@ -117,10 +117,10 @@
     label = "Simulation Preliminary"
 
 if "ttdilep" in args.phase:
-    input_txt = "dilepton ttbar"
+    input_txt = r"dileptonic t$\bar{t}$"
     nj = 2
 elif "ttsemilep" in args.phase:
-    input_txt = "semileptonic ttbar"
+    input_txt = r"t$\bar{t}$ $\mu$+jets"
```

Comment:
Not sure whether we need to add additional flag for input_txt for `cSF` related phase space since we also have the `ctag_ttdilep_sf` and `ctag_ttsemilep_sf`  for the phase space derive charm SFs.

Reply:
maybe one can add an additional text like `ctag` of ctagSF` or similar in case the chosen phase space starts with `ctag_tt`?